app/views/static/index.html.erb

```diff
@@ -136,33 +136,25 @@
             <div class="tesimonial">
               <img src="img/diaper_small.png" title="Diaperbase Logo" alt="Diaperbase Logo">
               <!-- <span class="name">Mashable</span> -->
-              <p>Diaperbase is great, it saves me time and other stuff.</p>
-              <span class="author">Rachel Alston - PDX Diaper Bank</span>
+              <p>Finally, an inventory management system designed specifically for the unique needs of diaper banks!</p>
+              <span class="author">Rachel Alston, PDX Diaper Bank</span>
             </div>
 
             <div class="tesimonial">
               <img src="img/diaper_small.png" title="Diaperbase Logo" alt="Diaperbase Logo">
               <!-- <span class="name">Mashable_2</span> -->
-              <p>Wow! It is amazingly easy to use.</p>
-              <span class="author">Sean Marica - Diaper Wearer</span>
+              <p>DiaperBase has taken our diaper distribution to the next level. We no longer have to worry about how to track our incoming and outgoing inventory, and having all our products and data tracked in different ways in different documents and systems. DiaperBase has saved us time, money, energy and many, many headaches. The system is extremely intuitive and user-friendly and we are so grateful to have it.</p>
+              <span class="author">Megan, Sweet Cheeks Diaper Bank</span>
             </div>
 
-            <div class="tesimonial">
-              <img src="img/diaper_small.png" title="Diaperbase Logo" alt="Diaperbase Logo">
-              <!-- <span class="name">Mashable_3</span> -->
-              <p>I don't always wear diapers but when I do, I get them from a diaper bank.</p>
-              <span class="author">Aaron Hill - Diaper Aficionado</span>
```

Comment:
I think people need to know that Aaron is a Diaper Aficionado... Can't we work this in somewhere?

Reply:
"All of the diapers I wear are provided by Diaperbase"